dev.aunetx.deezer.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>dev.aunetx.deezer</id>
+  <provides>
+    <id>dev.aunetx.deezer.desktop</id>
+  </provides>
+  <launchable type="desktop-id">dev.aunetx.deezer.desktop</launchable>
+  <name>Deezer</name>
+  <developer_name>aunetx</developer_name>
+  <summary>Online music streaming service</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.deezer.com/legal/cgu/</project_license>
+  <url type="homepage">https://www.deezer.com</url>
```

Comment:
given that it is not developed by deezer, I feel this shouldn't be here. it's murky.

Reply:
Yeah I did not really know, so just a link to https://github.com/aunetx/deezer-linux I guess ?